### PR TITLE
Auditors timestamps

### PR DESCRIPTION
**Pull request by budang**
Added timestamps to appropriate auditors and recreated the migrations

### CODE REVIEW

File: web/turkey/survey/static/survey/js/auditors/keypresses_specific.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 var AuditorKeypressesSpecific = {
+    start_date: new Date(),
     keypresses_specific: [],
     log_keypress_content: function (e) {
-        this.keypresses_specific.push({ 'key' : String.fromCharCode(e.which) });
+        this.keypresses_specific.push({
+            'key' : String.fromCharCode(e.which),
+            'time': (new Date()).getTime() - this.start_date.getTime()
```
Comment (CuriousG102) [inline web/turkey/survey/static/survey/js/auditors/keypresses_specific.js]: Unclosed parentheses?

Comment (CuriousG102) [inline web/turkey/survey/static/survey/js/auditors/keypresses_specific.js:7]: Nit: The parentheses around new Date() are unnecessary.

File: web/turkey/survey/static/survey/js/auditors/mouse_movement_specific.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 var AuditorMouseMovementSpecific = {
+    start_date: new Date(),
     mouse_movement_specific: [],
     log_mousemove_specific: function (e) {
         this.mouse_movement_specific.push(
             {
                 'x' : e.pageX,
-                'y' : e.pageY
+                'y' : e.pageY,
+                'time': (new Date()).getTime() - this.start_date.getTime()
```
Comment (CuriousG102) [inline web/turkey/survey/static/survey/js/auditors/mouse_movement_specific.js:9]: Nit: Same thing here

File: web/turkey/survey/static/survey/js/auditors/mouse_movement_specific.js [hunk 3]
```diff
@@ -26,4 +28,4 @@ $(window).mousemove(
 turkey.register_auditor('mouse_movement_specific',
                           auditor_mouse_movement_specific
                             .submit_callable
-                            .bind(auditor_mouse_movement_specific));
+                            .bind(auditor_mouse_movement_specific));
```
Comment (CuriousG102) [inline web/turkey/survey/static/survey/js/auditors/mouse_movement_specific.js:31]: Nit: Stylistically you want a newline at the end of a file, or we're going to get a spurious "changed line" whenever you add to this.

File: web/turkey/survey/static/survey/js/auditors/on_focus_time.js [hunk 4]
```diff
@@ -6,15 +6,15 @@ var AuditorOnFocusTime = {
     log_on_focus_time: function (e) {
         if(document[this.hidden]) {
             var focus_change_time = (new Date()).getTime();
-            this.on_focus_time += focus_change_time - this.last_focus_time;  
+            this.on_focus_time += focus_change_time - this.last_focus_time;
         } else {
-            this.last_focus_time = (new Date()).getTime(); 
+            this.last_focus_time = (new Date()).getTime();
```
Comment (CuriousG102) [inline web/turkey/survey/static/survey/js/auditors/on_focus_time.js:11]: Same thing here

File: web/turkey/survey/static/survey/js/auditors/recorded_time_disparity.js [hunk 5]
```diff
@@ -6,9 +6,9 @@ var AuditorRecordedTimeDisparity = {
     log_recorded_time_disparity: function (e) {
         if(document[this.hidden]) {
             var focus_change_time = (new Date()).getTime();
-            this.on_focus_time += focus_change_time - this.last_focus_time.getTime();  
+            this.on_focus_time += focus_change_time - this.last_focus_time.getTime();
         } else {
-            this.last_focus_time = (new Date()); 
+            this.last_focus_time = (new Date());
```
Comment (CuriousG102) [inline web/turkey/survey/static/survey/js/auditors/recorded_time_disparity.js:11]: Gratuitous parentheses

File: web/turkey/survey/static/survey/js/auditors/user_agent.js [hunk 6]
```diff
@@ -8,7 +8,7 @@ var AuditorUserAgent = {
         // var url = "http://www.useragentstring.com/?uas="
         //     + uas
         //     + "&getJSON=all";
-        
+
```
Comment (CuriousG102) [inline web/turkey/survey/static/survey/js/auditors/user_agent.js]: While we're modifying this, can we please get rid of the commented out code?

**Review by CuriousG102 — APPROVED** [review]
Overall looks fine to me. If we could take care of the nits before merging that would be preferable. At some point in the near future I think it may be worth it for us to have a code linter. But I'm not holding it up for those nits.

**Comment by CuriousG102** [comment]
Merge conflict after merging #31

**Comment by budang** [comment]
@CuriousG102 parenthesis and line breaks were stylistic preferences from several months ago, just keeping consistency

can keep as is or go back and change for all files, whichever you prefer

File: web/turkey/survey/static/survey/js/auditors/user_agent.js (hunk 6 above)
Comment (budang) [reply web/turkey/survey/static/survey/js/auditors/user_agent.js]: sure

**Comment by CuriousG102** [comment]
Ok. Don’t care too much and I believe you if I flip flopped on it. Probably just a function of familiarity breeding comfort with lack of parentheses being fine.

Miles Hutson
(214) 929-5684

**Pull request merged**